gradle/libs.versions.toml

```diff
@@ -1,5 +1,6 @@
 [versions]
 awsSdkVersion = { prefer = '1.12.544' }
+awsSdk2 = {prefer = '2.26.1'}
```

Comment:
Next PR will use this library for presigning s3 url instead of sdk 1. Had to revert changes because mockito in GitHub had a disagreement with the library.